test/unittest/blas1/blas1_rotmg_test.cpp

```diff
@@ -129,8 +140,7 @@ void RotmgTest<scalar_t, mem_alloc>::validate_with_reference() {
 
   const bool isAlmostEqual = utils::almost_equal(sycl_out.d1, d1_ref) &&
                              utils::almost_equal(sycl_out.d2, d2_ref) &&
-                             utils::almost_equal(sycl_out.x1, x1_ref) &&
-                             utils::almost_equal(sycl_out.y1, y1_ref);
+                             utils::almost_equal(sycl_out.x1, x1_ref);
```

Comment:
According to oneMKL interface specification y1 is not an output, so I removed from the validation. Moreover, given the new interface using scalars, the value is not copied out and it is lost after computation.